osu.Game/Overlays/Direct/DirectRulesetSelector.cs

```diff
@@ -26,8 +26,13 @@ public DirectRulesetSelector()
             TabContainer.Masking = false;
             TabContainer.Spacing = new Vector2(10, 0);
             AutoSizeAxes = Axes.Both;
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
 
-            Current.DisabledChanged += value => SelectedTab.FadeColour(value ? Color4.DarkGray : Color4.White, 200, Easing.OutQuint);
+            Current.BindDisabledChanged(value => SelectedTab.FadeColour(value ? Color4.DarkGray : Color4.White, 200, Easing.OutQuint));
```

Comment:
probably want a `,true` to handle the case it is disabled before `LoadComplete`

Reply:
WHOOPS